packages/client/hmi-client/src/components/workflow/ops/model-comparison/model-comparison-operation.ts

```diff
@@ -5,6 +5,7 @@ const DOCUMENTATION_URL =
 	'https://githubicom/gyorilab/mira/blob/7314765ab409ddc9647269ad2381055f1cd67706/notebooks/hackathon_2023.10/dkg_grounding_model_comparison.ipynb#L307';
 
 export interface ModelComparisonOperationState extends BaseState {
+	overview: string;
```

Comment:
We should avoid storing giant strings, or large data structures in operator state. It slow things down and makes all other workflow related tasks more difficult to coordinate.

Reply:
for this pass, I am actually not going to save anything into the state. I changed it to an `overviewId` but not gonna use it until I chat with @kbirk about how we should cache them